backend/src/activitypub/objects/note.ts

```diff
@@ -41,8 +41,8 @@ export async function createPublicNote(
 		sensitive: false,
 		summary: null,
 		tag: [],
-		attachment,
 
+		attachment: attachments,
```

Comment:
@xtuc in your comment: https://github.com/cloudflare/wildebeest/pull/152/files#r1085391616
are you saying that here we should take the attachments and replace all types `image` and `video` with `Image` and `Video`? (`audio` and `Audio` too I guess?)

Reply:
Not applied as it is not the right solution